tools/test/votest_helper.py

```diff
@@ -0,0 +1,205 @@
+    tests = get_config(data=data, vo=vo, section="tests")
+    allowed_paths = set(filter_paths(combine_paths(resolve_paths(resolve_path_keywords(tests.get('allow', []))))))
+    excluded_paths = set(filter_paths(combine_paths(resolve_paths(resolve_path_keywords(tests.get('deny', []))))))
+    tests_to_run = allowed_paths - excluded_paths
+    return " ".join(tests_to_run)
+
+
+def build_config(input_conf):
+    build_matrix: list = None
+    try:
+        build_matrix = [
+            {
+                "POLICY": policy_package,
+                "DIST": dist,
+                "RDBMS": rdbms,
+                "PYTHON": python_ver,
+                "SUITE": "votest",
+                "IMAGE_IDENTIFIER": f"votest-{image_identifier}",
+            }
+            for policy_package in input_conf
+            for dist in input_conf[policy_package]['dists']
+            for rdbms in input_conf[policy_package]['rdbms']
+            for image_identifier in input_conf[policy_package]['image_identifier']
+            for python_ver in input_conf[policy_package]['python']
+        ]
+    except KeyError as e:
+        logger.warning(f"Key not found for policy package. Check YAML schema. Details: {e}")
+        sys.exit(1)
+    return json.dumps(build_matrix)
+
+
+def load_config_file(policy_package_matrix_file):
+    input_conf: list = None
+    with open(policy_package_matrix_file, 'r') as stream:
+        try:
+            input_conf = yaml.safe_load(stream)
+        except yaml.parser.ParserError:
+            traceback.print_exc()
+            logger.warning("Error parsing matrix for policy packages. Invalid YAML syntax")
+            sys.exit(1)
+    return input_conf
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Utils for extracting information from policy package build matrix')
+    parser.add_argument('--file',
+                        metavar="file",
+                        type=lambda p: Path(p).absolute(),
+                        default=Path(Path(__file__).parent.parent.parent / "etc" / "docker" / "test" / "matrix_policy_package_tests.yml").absolute(),
+                        help='the path to matrix_policy_package_tests.yml',
+                        )
+    parser.add_argument('--rucio-cfg',
+                        metavar="rucio",
+                        type=lambda p: Path(p).absolute(),
+                        default=Path(Path(__file__).parent.parent.parent / "etc" / "rucio.cfg").absolute(),
+                        help='the path to rucio.cfg',
+                        )
+    parser.add_argument('--vo', type=str, default='all', required=False)
+    parser.add_argument('-v', action='store_true',
+                        required=False,
+                        help="Verbose mode to show logged errors")
+    output_type = parser.add_mutually_exclusive_group(required=False)
+    output_type.add_argument('-i', required=False,
```

Comment:
This name is not descriptive. How about `--return-policy-package-installation-command`, or `--policy-package-installation-command`?